docs/sdks/remove-api-key.md

```diff
@@ -32,10 +32,23 @@ client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                         email='nemo@nautilus.com', password='xxxxx',
                         geofabric='_system')
 
-# Remove api key
-remove = client.remove_api_key('id1')
+# Check wether the chosen_api_key exists or not, comparing it to the list elements
```

Comment:
```suggestion
# Checks if `chosen_api_key` exists
```

What do we mean by "comparing it to the list elements"?

Reply:
Sorry, I missed adding a few lines of code. Fixed.

A list of all API key IDs a user has is saved in a variable which is used to make a comparison check later.

Reply:
I have updated the comment to improve semantics.